classes/class-list-table.php

```diff
@@ -718,55 +718,46 @@ public function filters_form() {
 	}
 
 	public function filter_select( $name, $title, $items, $ajax = false ) {
-		if ( $ajax ) {
-			$out = sprintf(
-				'<input type="hidden" name="%s" class="chosen-select" value="%s" data-placeholder="%s" />',
-				esc_attr( $name ),
-				esc_attr( wp_stream_filter_input( INPUT_GET, $name ) ),
-				esc_attr( $title )
-			);
-		} else {
-			$options  = array( '<option value=""></option>' );
-			$selected = wp_stream_filter_input( INPUT_GET, $name );
-
-			foreach ( $items as $key => $item ) {
-				$value       = isset( $item['children'] ) ? 'group-' . $key : $key;
-				$option_args = array(
-					'value'    => $value,
-					'selected' => selected( $value, $selected, false ),
-					'disabled' => isset( $item['disabled'] ) ? $item['disabled'] : null,
-					'icon'     => isset( $item['icon'] ) ? $item['icon'] : null,
-					'group'    => isset( $item['children'] ) ? $key : null,
-					'tooltip'  => isset( $item['tooltip'] ) ? $item['tooltip'] : null,
-					'class'    => isset( $item['children'] ) ? 'level-1' : null,
-					'label'    => isset( $item['label'] ) ? $item['label'] : null,
-				);
-				$options[]   = $this->filter_option( $option_args );
-
-				if ( isset( $item['children'] ) ) {
-					foreach ( $item['children'] as $child_value => $child_item ) {
-						$option_args = array(
-							'value'    => $child_value,
-							'selected' => selected( $child_value, $selected, false ),
-							'disabled' => isset( $child_item['disabled'] ) ? $child_item['disabled'] : null,
-							'icon'     => isset( $child_item['icon'] ) ? $child_item['icon'] : null,
-							'group'    => $key,
-							'tooltip'  => isset( $child_item['tooltip'] ) ? $child_item['tooltip'] : null,
-							'class'    => 'level-2',
-							'label'    => isset( $child_item['label'] ) ? '- ' . $child_item['label'] : null,
-						);
-						$options[]   = $this->filter_option( $option_args );
-					}
+		$options  = [ '<option value=""></option>' ];
+		$selected = wp_stream_filter_input( INPUT_GET, $name );
+
+		foreach ( $items as $key => $item ) {
+			$value       = isset( $item['children'] ) ? 'group-' . $key : $key;
+			$option_args = [
```

Comment:
Can you remove these short array syntax change please, not sure what Stream's preference is here, so we should not change code unrelated to the PR purpose